index.js

```diff
@@ -3657,14 +3661,29 @@ export const createContainerSpecLikeBom = async (path, options) => {
   }
   // Privado.ai json files
   const privadoFiles = getAllFiles(path, ".privado/" + "*.json");
-  // parse yaml manifest files
-  if (dcFiles.length) {
-    for (const f of dcFiles) {
+
+  // Parse yaml manifest files or dockerfiles
+  if (dcFiles.length || dfFiles.length) {
+    for (const f of [...dcFiles, ...dfFiles]) {
       if (DEBUG_MODE) {
         console.log(`Parsing ${f}`);
       }
-      const dcData = readFileSync(f, { encoding: "utf-8" });
-      const imglist = parseContainerSpecData(dcData);
+      const dData = readFileSync(f, { encoding: "utf-8" });
+      let imglist = [];
+      // parse yaml manifest files
+      if (f.endsWith(".yml") || f.endsWith(".yaml")) {
+        imglist = parseContainerSpecData(dData);
+      } else {
+        // dockerfiles
+        for (const dfLine of dData.split("\n")) {
+          if (dfLine.includes("FROM")) {
+            imglist.push({
+              image: dfLine.split("FROM")[1].split("AS")[0].trim()
```

Comment:
Do we need all the FROM statements or only the last one?

Reply:
We use a lot of mutli-stage Docker builds and I'd personally be interested in seeing each image used.